loops/inner_loop.py

```
@@ -796,6 +828,39 @@ def _build_reset_pr(existing_pr: Optional[RunPR]) -> Optional[RunPR]:
     )
 
 
+def _load_runtime_config(
+    *,
+    run_dir: Path,
+    run_log: Path,
+) -> Optional[InnerLoopRuntimeConfig]:
+    try:
+        return read_inner_loop_runtime_config(run_dir)
+    except Exception as exc:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)